.github/CODEOWNERS

```
@@ -1,4 +1,4 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-* @deluca-mike @sammy1991106 @re1ro @Foivos
+* @deluca-mike @sammy1991106 @re1ro @Foivos @milapsheth
```

```suggestion
* @sammy1991106 @re1ro @Foivos @milapsheth
```
removed deluca-mike since he's not under the org anymore, so github is complaining